http-ballerina/src/http-tests/tests/unit-tests/services/dispatching/data-binding-test.bal

```diff
@@ -23,11 +23,6 @@ import http;
 listener http:Listener dataBindingEP = new(databindingTest);
 http:Client dataBindingClient = new("http://localhost:" + databindingTest.toString());
 
-type Person record {|
```

Comment:
Why did we remove this? Do we have similar record somewhere else?

Reply:
Yes, we use `[here](https://github.com/ballerina-platform/module-ballerina-http/pull/27/files#diff-f4739da12d38d76e8ee77d534dea15ccR23)` as well. I have renamed that.